developer-documentation/500-assets/media-selection.md

```diff
@@ -32,3 +32,12 @@
   * leftBottom (default: true)
   * bottom (default: true)
   * rightBottom (default: true)
+ 
+## Get displayOption on Website Template
```

Comment:
I would favour a variant where you display all the values from the view in a table, we can't add an own heading for every value.